packages/cli-platform-android/src/commands/buildAndroid/startServerInNewWindow.ts

```diff
@@ -17,19 +21,16 @@ export function startServerInNewWindow(
     : 'launchPackager.command';
   const packagerEnvFilename = isWindows ? '.packager.bat' : '.packager.env';
   const portExportContent = isWindows
-    ? `set RCT_METRO_PORT=${port}`
-    : `export RCT_METRO_PORT=${port}`;
+    ? `set RCT_METRO_PORT=${port}\nset PROJECT_ROOT=${projectRoot}`
+    : `export RCT_METRO_PORT=${port}\nexport PROJECT_ROOT=${projectRoot}`;
 
   /**
    * Set up the `.packager.(env|bat)` file to ensure the packager starts on the right port.
    */
-  const launchPackagerScript = path.join(
-    reactNativePath,
-    `scripts/${scriptFile}`,
-  );
+  const launchPackagerScript = path.join(reactNativeCliPath, `${scriptFile}`);
 
   /**
-   * Set up the `launchpackager.(command|bat)` file.
+   * Set up the `launchPackager.(command|bat)` file.
    * It lives next to `.packager.(bat|env)`
```

Comment:
Yeah, I'm also not fan of this. That's good idea but on the other hand the problem is `.packager.env` file right now lives next to `launchPackager.sh` so we're 100% sure about file path and we can make `source "$THIS_DIR/.packager.env"`. But when we move `.packager.env` to e.g. `packages/app-one/node_modules/.packager.env` and we try to run the script which is e.g. in `node_modules/react-native-community/cli-plugin-metro/launchPackager.sh` we can't source the `.packager.env` file because we don't its path. 😟

Reply:
Oof, yeah… Can we generate `node_modules/.bin/launchPackager.sh` instead and pass that script to `open`? That way we will always know where the project root is.